web/src/app/users/UserContactMethodListDest.tsx

```diff
@@ -0,0 +1,265 @@
+    ]
+
+    // disable send test and reactivate if not current user
+    if (!cm.disabled) {
+      actions.push({
+        label: 'Send Test',
+        onClick: () => setShowSendTestByID(cm.id),
+        disabled: !isCurrentUser,
+        tooltip: !isCurrentUser
+          ? 'Send Test only available for your own contact methods'
+          : '',
+      })
+    } else {
+      actions.push({
+        label: 'Reactivate',
+        onClick: () => setShowVerifyDialogByID(cm.id),
+        disabled: !isCurrentUser,
+        tooltip: !isCurrentUser
+          ? 'Reactivate only available for your own contact methods'
+          : '',
+      })
+    }
+    return actions
+  }
+
+  function getSecondaryAction(cm: UserContactMethod): JSX.Element {
+    return (
+      <Grid container spacing={2} alignItems='center' wrap='nowrap'>
+        {cm.disabled && !props.readOnly && !mobile && isCurrentUser && (
+          <Grid item>
+            <Button
+              aria-label='Reactivate contact method'
+              onClick={() => setShowVerifyDialogByID(cm.id)}
+              variant='contained'
+            >
+              Reactivate
+            </Button>
+          </Grid>
+        )}
+        {!props.readOnly && (
+          <Grid item>
+            <OtherActions actions={getActionMenuItems(cm)} />
+          </Grid>
+        )}
+      </Grid>
+    )
+  }
+
+  function getSubText(cm: UserContactMethod): JSX.Element {
+    return (
+      <React.Fragment>
+        {cm.dest.values.map((v) => {
+          let cmText = v.label
+          if (cm.pending) {
+            cmText = `${cmText} - this contact method will be automatically deleted if not verified`
+          }
+          if (v.fieldID === 'webhook-url') {
+            return (
+              <Typography key={v.toString()}>
+                {`${cmText} (`}
+                <AppLink to='/docs#webhooks'>docs</AppLink>)
+              </Typography>
+            )
+          }
+
+          return <Typography key={v.toString()}>{cmText}</Typography>
+        })}
+      </React.Fragment>
+    )
+  }
+
+  return (
+    <Grid item xs={12}>
+      <Card>
+        <CardHeader
+          className={classes.cardHeader}
+          titleTypographyProps={{ component: 'h2', variant: 'h5' }}
+          title='Contact Methods'
+          action={
+            !mobile ? (
+              <Button
+                title='Create Contact Method'
+                variant='contained'
+                onClick={() => setShowAddDialog(true)}
+                startIcon={<Add />}
+              >
+                Create Method
+              </Button>
+            ) : null
+          }
+        />
+        <FlatList
+          data-cy='contact-methods'
+          items={sortContactMethods(contactMethods).map((cm) => ({
+            title: `${cm.name} (${cm.dest.type})${cm.disabled ? ' - Disabled' : ''}`,
+            subText: getSubText(cm),
+            secondaryAction: getSecondaryAction(cm),
+            icon: getIcon(cm),
```

Comment:
Should we update this to be a `<React.Fragment>` that starts with the `DestinationAvatar`?

Since we don't have icons for contact method types, maybe that should be out of scope for this PR, but probably necessary before taking the flag out of the experimental status.